platform/platform-impl/src/com/intellij/ide/actions/SendFeedbackAction.java

```diff
@@ -43,11 +62,11 @@ public static void launchBrowser(@Nullable Project project) {
       .replace("$BUILD", eap ? appInfo.getBuild().asStringWithoutProductCode() : appInfo.getBuild().asString())
       .replace("$TIMEZONE", System.getProperty("user.timezone"))
       .replace("$EVAL", isEvaluationLicense() ? "true" : "false")
-      .replace("$DESCR", getDescription());
+      .replace("$DESCR", getDescription(project));
     BrowserUtil.browse(urlTemplate, project);
   }
 
-  public static String getDescription() {
```

Comment:
It was a public static method with possible usages from plugins. It's better to wrap it with delegation to `getDescription(null);` instead of removing